nbs/tutorials/07-forest_plot.ipynb

```diff
@@ -0,0 +1,681 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cf1612f8",
+   "metadata": {},
+   "source": [
+    "# Forest Plot\n",
+    "\n",
+    "> Explanation of how to use forest_plot to compare contrast objects such as delta-delta and mini-meta using DABEST.\n",
```

Comment:
Can change to a simpler wording. This subsection takes too much space in the first glance